### DISCUSSION TITLE
Ideation for DIVE package

### DISCUSSION THREAD

**Discussion by Mr-Nobody21** [HugoByte/DIVE#26] [Ideas]
Ideation for DIVE package 
all ideas are welcomed

**Comment by CyrusVorwald** [comment]
Connecting to the development tools that run their own node, such as hardhat, foundry, truffle, and drogon

**Comment by hemz10** [comment]
Would like to have a custom successful or failure log printed on output when verbose is enabled. So that we can know the status of our execution.

**Comment by hemz10** [comment]
How about adding time taken for execution along with current implementation of total and current steps. So that we will have an idea of how long it will take for a bridge setup or chain to spin up.

**Comment by hemz10** [comment]
Also would like to have timeout for execution. Sometimes execution gets stuck in same step for long time. I had to wait thinking that execution will go to next step but it never did. So having a timeout and failing if it reaches the time limit and removing all service related artifacts will save some time. And also removing artifacts can help in restarting the node without cleaning entire enclave.
